src/main/java/sg/ncl/MainController.java

```diff
@@ -145,19 +146,21 @@ public String calendar1() {
     }
 
     @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
-    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException {
+    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+        InputStream stream = null;
         response.setContentType("application/pdf");
         try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this "try" to a try-with-resources. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2093)